posthog/queries/breakdown_props.py

```diff
@@ -160,12 +178,20 @@ def _to_value_expression(
     breakdown: Union[str, List[Union[str, int]], None],
     breakdown_group_type_index: Optional[GroupTypeIndex],
     direct_on_events: bool = False,
+    cast_as_float: bool = False,
 ) -> str:
-    if breakdown_type == "person":
-        return get_single_or_multi_property_string_expr(
+    if breakdown_type == "session":
+        if breakdown == "$session_duration":
+            # Return the session duration expression right away because it's already an number,
+            # so it doesn't need casting for the histogram case (like the other properties)
+            return f"{SessionQuery.SESSION_TABLE_ALIAS}.session_duration as value"
```

Comment:
is there a reason why we wouldn't want to cast this to float64 or null? Would be nice to not special case this and keep things uniform

Reply:
I thought our property extractions stuff always returns a string. Looking more now

Reply:
Ah! Probably. So maybe this matters only for calculated properties, a.k.a. just duration.

I made them uniform anyway